packages/addons/service/syncthing/package.mk

```diff
@@ -0,0 +1,87 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016 Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="syncthing"
+PKG_VERSION="0.12.22"
+PKG_REV="100"
+PKG_ARCH="any"
+PKG_LICENSE="MPLv2"
+PKG_SITE="https://syncthing.net/"
+PKG_URL="https://github.com/syncthing/syncthing/archive/v${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_TARGET="toolchain go:host"
+PKG_PRIORITY="optional"
+PKG_SECTION="service/system"
+
+PKG_SHORTDESC="Open Source Continuous File Synchronization"
+PKG_LONGDESC="Syncthing replaces proprietary sync and cloud services with something open, trustworthy and decentralized. Your data is your data alone and you deserve to choose where it is stored, if it is shared with some third party and how it's transmitted over the Internet."
+PKG_MAINTAINER="Anton Voyl (awiouy at gmail dot com)"
+PKG_ADDON_REPOVERSION="8.0"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_TYPE="xbmc.service"
+PKG_ADDON_PROVIDES=""
+PKG_AUTORECONF="no"
+
+configure_target() {
+
+  case $TARGET_ARCH in
+    x86_64)
+      export GOARCH=amd64
+      ;;
+    arm)
+      export GOARCH=arm
+
+      case $TARGET_CPU in
+        arm1176jzf-s)
+          export GOARM=6
+          ;;
+        cortex-a7)
```

Comment:
can you make this line `cortex-a7|cortex-a9)` so we can compile this for WeTek_Core and WeTek_Play.